src/plugins/event-plugins/DomEventPlugin.ts

```diff
@@ -129,13 +129,30 @@ export class DomEventPlugin implements Plugin {
             };
             if (cssLocator !== undefined) {
                 eventData.cssLocator = cssLocator;
+                const elementRumId = this.getElementRumId(event, cssLocator);
```

Comment:
question: Why restrict this to css locators? For example, if I record all clicks on the `document` element, I may still want to use data-rum-id

Reply:
Generalizing this would be even more preferable. In this case we could look up for the `data-rum-id` attribute to locate the `firstMatchingElement`.